app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-ecommerce-tracker.js

```diff
@@ -1,4 +1,4 @@
-//= require govuk/vendor/polyfills/Element/prototype/closest.js
+//= require ../vendor/polyfills-govuk-frontend-v4/Element/prototype/closest.js
```

Comment:
This probably doesn't matter since we'll be deleting the polyfills in the future soon anyway, but in the commit that adds polyfills there are loads of polyfills (plus some `map` files, do we need those?) but in this commit it looks like the components only use a small handful of them. What are the extra polyfills for?